params/governance_params.go

```diff
@@ -94,6 +96,7 @@ var (
 	DefaultBaseFeeDenominator        = uint64(20)
 	DefaultMintingAmount             = big.NewInt(0)
 	DefaultRatio                     = "100/0/0"
+	DefaultKip82Ratio                = "20/80"
```

Comment:
Can we use "100/0" as default? just to match with `DefaultRatio = "100/0/0"`.

Reply:
fixed in 8c6e44e1

Reply:
I'll revert this change, because it will be directly affected to Cypress at Kore hardfork block.

Reply:
reverted in 0b3e45e2

Reply:
Thank you!